comp/otelcol/otlp/components/exporter/logsagentexporter/factory.go

```diff
@@ -42,6 +43,21 @@ type Config struct {
 
 	// HostMetadata defines the host metadata specific configuration
 	HostMetadata datadogconfig.HostMetadataConfig `mapstructure:"host_metadata"`
+
+	OrchestratorConfig OrchestratorConfig
+}
+
+// OrchestratorConfig contains configuration for sending orchestrator data to Datadog.
+type OrchestratorConfig struct {
+	// ClusterName is the name of the Kubernetes cluster to associate with the orchestrator data.
+	ClusterName string
```

Comment:
To match with this PR https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/8a72bd993bdd340737a700acd783e578d9c69059/pkg/datadog/config/orchestrator_explorer.go#L9-L15

Let's add `Enabled` field as well ?

When routing the logs, we check both scope name and `Enabled`